assets/pages/new-contributions-landing/components/SetPassword/helpers.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+// ----- Imports ----- //
+import { routes } from 'helpers/routes';
+import { logException } from 'helpers/logger';
+
+import type { Csrf as CsrfState } from 'helpers/csrf/csrfReducer';
+import { logPromise } from 'helpers/promise';
+
+// ----- Functions ----- //
+
+const requestData = (password: string, guestAccountRegistrationToken: string, csrf: CsrfState) => ({
```

Comment:
There is a similar function in `paymentApis` that you can perhaps generalise

Reply:
I've now moved the `setPasswordGuest` function to `readerRevenuesApis` and deleted the `requestData` function, instead using a generalised `requestOptions` function (previously `postRequestOptions`